### PR TITLE
TehanovAnton/hometask-5

### PR DESCRIPTION
**Pull request by TehanovAnton**

### CODE REVIEW

File: TehanovAnton/hometask-5/hometask [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+2) select f.title, l.name
+	from film f inner join language l on f.language_id = l.language_id 
+	where l.name = 'English' or l.name = 'German' or l.name = 'Italian';
+
+3) select first_name || '-' || last_name
+	from actor
+	where first_name like 'W%' or last_name like '%h';
+
+4) select rental_duration, count(rental_duration) count
```
Comment (dmokreckiy) [inline TehanovAnton/hometask-5/hometask:11]: Please sort it by `rental_duration` in ascending order.

File: TehanovAnton/hometask-5/hometask [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+	from film
+	group by rental_duration;
+
+5) select l.name as language, count(f.title)    
+	from language l inner join film f on l.language_id = f.language_id 
+	group by l.name;
```
Comment (dmokreckiy) [inline TehanovAnton/hometask-5/hometask:18]: image
And other languages?

File: TehanovAnton/hometask-5/hometask [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  where c.name = 'Music');
+
+delete from film_category where category_id in (
+  select category_id
+	from category where name = 'Music');
+
+delete from film f
+where (select count(*) from film_category fc where fc.film_id = f.film_id) = 0;
+
+9) select a.first_name
```
Comment (dmokreckiy) [inline TehanovAnton/hometask-5/hometask]: Please add `last_name`

**Review by dmokreckiy — CHANGES_REQUESTED** [review]
Please see my comments below

File: TehanovAnton/hometask-5/hometask (hunk 2 above)
Comment (TehanovAnton) [reply TehanovAnton/hometask-5/hometask:18]: All films are in English. There are no films in other languages.

File: TehanovAnton/hometask-5/hometask_report.md [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+
+```
+SELECT l.name as language, count(f.title)    
+FROM language l INNER JOIN film f on l.language_id = f.language_id 
+GROUP BY l.name;
+```
+```
+       language       | count 
+----------------------+-------
+ English              |   949
```
Comment (dmokreckiy) [inline TehanovAnton/hometask-5/hometask_report.md:100]: You should have re-seeded DB from dump after deletion. But still valid.

**Review by dmokreckiy — APPROVED** [review]
LGTM
Kudos for markdown report.

**Pull request merged**